src/utilities/isNoFlowFileAnnotation.js

```diff
@@ -0,0 +1,17 @@
+import _ from 'lodash';
+
+const FLOW_MATCHER = /^@noflow$/;
+
+export default (comment, strict) => {
+  // The flow parser splits comments with the following regex to look for the @flow flag.
+  // See https://github.com/facebook/flow/blob/a96249b93541f2f7bfebd8d62085bf7a75de02f2/src/parsing/docblock.ml#L39
+  return _.some(comment.split(/[\t\n\r */\\]+/), (commentPart) => {
+    const match = commentPart.match(FLOW_MATCHER);
+
+    if (match === null) {
+      return false;
+    }
+
+    return !strict || match[0] === '@noflow';
```

Comment:
This file is a copy of "isFlowFileAnnotation.js" with only changes in regex and on this line